### DISCUSSION TITLE
Add support for Brothers in Arms - Earned in Blood

### DISCUSSION THREAD

**Discussion by cmbasnett** [gildor2/UEViewer#272] [Ideas]
UEViewer is not currently able to open packages from [Brothers in Arms - Earned in Blood](https://en.wikipedia.org/wiki/Brothers_in_Arms:_Earned_in_Blood).

![image](https://user-images.githubusercontent.com/5035660/172453923-b4c43387-f9e2-4da2-864c-b2808d089591.png)

Here are a couple of packages that do not open:

[k_character_us.zip](https://github.com/gildor2/UEViewer/files/8855611/k_character_us.zip)

**Comment by GregorHecker** [comment]
Hi Basnett, i am Hecker, well i have the same problem, but i have a answer because cann't open the BIA: RTH30 and EIB files.
Its easy BIA: EIB and RTH30 use UE2 but that version was modded by developers (Unreal Engine 2.5) and well the files cann't open in both cases the result gonna be the same.
I am waiting for a new version or a Viewer more simple and compatible with UE 2.5 version

**Reply by GregorHecker** [reply]
In gildor website have more info about the problem

https://www.gildor.org/smf/index.php?topic=6514.0
